sdk/storage/storage-blob-changefeed/rollup.base.config.js

```diff
@@ -165,8 +165,8 @@ export function browserConfig(test = false) {
     ];
     baseConfig.plugins.unshift(multiEntry({ exports: false }));
     baseConfig.output.file = "dist-test/index.browser.js";
-    // mark fs-extra as external
-    baseConfig.external = ["fs-extra"];
+
+    baseConfig.external = [];
```

Comment:
Why do we need to set an empty array? :)

Reply:
A very silly reason... :)
Thought that the placeholder would be good to have when we do want to add something to the array since we already had the assignment statement.

Was about to clean it up, but then we already have the other effort of using a shared rollup config, so I didnt prioritize it much